NOTES.UNIX

```diff
@@ -8,35 +8,53 @@
  Shared libraries and installation in non-standard locations
  -----------------------------------------------------------
 
- Binaries on Unix variants expect to find shared libraries in standard
- locations, such as /usr/lib, /usr/local/lib and some other locations
- configured in the system (for example /etc/ld.so.conf on some systems).
- If the libraries are installed in non-standard locations, binaries
- will not find them and therefore fail to run unless they get a bit of
- help from a defined RPATH or RUNPATH.  This can be applied by adding
- the appropriate linker flags to the configuration command, such as
- this (/usr/local/ssl was the default location for OpenSSL installation
- in versions before 1.1.0):
+ Binaries on some Unix variants expect to find shared libraries in
+ standard locations, such as /usr/lib and possibly /usr/local/lib.  If
+ the libraries are installed in non-standard locations, binaries will not
+ find them and therefore fail to run unless they get a bit of help from
+ a defined runtime shared library search path.
+
+ It is possible to specify additonal directories for the runtime shared
+ library search path when linking a shared library or an application.
+ They are specified with different linking options depending on operating
+ system and versions thereof, and are talked about differently in their
+ respective documentation, where variations of RPATH are the most usual
+ (to be noted: GNU ld.so systems have two such tags, more on that below).
```

Comment:
Hmm... Those things are rather ELF-specific than GNU ld.so. At least Solaris has them, FreeBSD has them (and presumably remaining BSDs), ...

Reply:
Ah, thanks, I was unsure about that, as must pages I googled were talking about the RPATH/RUNPATH in terms of GNU ld.so specifically...